vyxal/elements.py

```diff
@@ -3193,6 +3194,8 @@ def n_choose_r(lhs, rhs, ctx):
             random.choice(lhs) for _ in range(abs(int(rhs)))
         ],
         (str, str): lambda: int(set(lhs) == set(rhs)),
+        (types.FunctionType, ts[1]): lambda: n_choose_r(rhs, lhs, ctx),
```

Comment:
Why is this one `n_choose_r` and not `drop_while`?

Reply:
`n_choose_r` is the parent function, so it doesn't really matter which, but I'll switch that around.

```suggestion
        (types.FunctionType, ts[1]): lambda: drop_while(rhs, lhs, ctx),
```